data/make_data.py

```
@@ -0,0 +1,160 @@
+########################################################################################################
+
+from tokenizer.rwkv_tokenizer import TRIE_TOKENIZER
+tokenizer = TRIE_TOKENIZER("tokenizer/rwkv_vocab_v20230424.txt")
+from src.binidx import MMapIndexedDataset
+def index_file_path(prefix_path):
+    return prefix_path + ".idx"
+def data_file_path(prefix_path):
+    return prefix_path + ".bin"
+class MMapIndexedDatasetBuilder(object):
+    def __init__(self, out_file, dtype=np.uint16):
+        self._data_file = open(out_file, "wb")
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)